core/platform/datastore/cloud_datastore_services.py

```diff
@@ -89,8 +90,10 @@ def get_multi(keys):
     return ndb.get_multi(keys)
 
 
-def update_timestamps_multi(entities, update_last_updated_time=True):
-    # type: (Sequence[base_models.BaseModel], bool) -> None
+def update_timestamps_multi(
+        entities: Sequence[base_models.BaseModel],
```

Comment:
I see backend errors failing. This is due to the fact that base_models is imported in if MYPY block. Either enclose this in quotes of import base_models outside the if MYPY block. Thanks!

Reply:
Fixed.